tools/rosgraph/src/rosgraph/roslogging.py

```diff
@@ -40,6 +40,8 @@
 import logging
 import logging.config
 
+import yaml
```

Comment:
I see it there : https://github.com/ros/ros_comm/blob/lunar-devel/tools/roslaunch/package.xml#L25